test/helpers/smartcards.ts

```diff
@@ -13,19 +15,185 @@ const contest0 = election.contests[0] as CandidateContest
 const contest1 = election.contests[1] as CandidateContest
 const contest0candidate0 = contest0.candidates[0]
 const contest1candidate0 = contest1.candidates[0]
-export const sampleVotes = vote(election.contests, {
-  president: [contest0candidate0],
-  'question-a': 'no',
-  'question-b': 'yes',
-  senator: [contest1candidate0],
-})
+const altBallotStyleId = election.ballotStyles[1].id
+const altPrecinctId = election.precincts[1].id
+
+export const sampleVotes0: VotesDict = vote(
+  getContests({
+    ballotStyle: election.ballotStyles[0],
+    election: election,
+  }),
+  {
+    president: [contest0candidate0],
+    'question-a': 'no',
+    'question-b': 'yes',
+    senator: [contest1candidate0],
+  }
+)
+
+export const sampleVotes1: VotesDict = vote(
+  getContests({
+    ballotStyle: election.ballotStyles[0],
+    election: election,
+  }),
+  {
+    '102': 'yes',
+    president: [
```

Comment:
If it's useful, this can just be `president: 'barchi-hallaren'`. Since you're passing the contests as the first parameter, `vote` will look up the candidate for you.